Doc/using/configure.rst

```diff
@@ -131,6 +131,17 @@ General Options
 
    .. versionadded:: 3.11
 
+.. cmdoption:: --enable-pystats
```

Comment:
https://github.com/python/cpython/commit/342b93f9f28746abb7b221a61d5a9b26ccbb395a says `--with-pystats`. Which one is correct?

Reply:
This is. @tiran requested that it was changed to `--enable-pystats`